frontend/AdbNav.vue

```diff
@@ -32,8 +32,8 @@
           <b-icon icon="account" size="is-small" class="mx-1"></b-icon>
           {{ user }} ({{ chapter }})
         </div>
-        <div class="buttons">
-          <a href="/logout" class="button is-light">Log out</a>
+        <div>
```

Comment:
does this still look like a button?

Reply:
https://buefy.org/documentation/button/ if you wanna make it smaller

jw if it looks bad not being a button... can you post a screenshot?

Reply:
I just updated it to color:inherit
![image](https://github.com/user-attachments/assets/88b87584-59c9-41c1-a6f9-1cfc56a59919)

A minute ago it was blue/purple:
![image](https://github.com/user-attachments/assets/e17b65a4-de56-4a5f-8d41-cfb30a6ee1c4)

Reply:
hmmm it looks kinda weird to me how similar it looks to the `Test User (SF Bay Area)` text. i think i prefer the other color if it's not a button tbh. or maybe the text could just be a little smaller or a different weight or something. not gonna block on it though.

Reply:
I made it blue (color: LinkText) since "visited" is probably not a helpful concept for a logout link.
![image](https://github.com/user-attachments/assets/ca7e29e5-f36c-4681-888c-4be0179a5a9a)